### ISSUE TITLE
Plan to release the code of DynIBaR?

### ISSUE THREAD

**Issue by whuzjw** [google-research/3d-moments#7]
Great Work! And I noticed that the enhanced version DynIBaR is also your work. I just want to ask if you have any plans to publish the code. Thanks in advance!

**Comment by snaves** [comment]
Yes, we plan to release the DynIBaR code soon, to this repo: https://github.com/google/dynibar -- this should happen before CVPR 2023!
